benchmarks/swe-bench/swe-bench/scripts/run_test.sh

```
@@ -0,0 +1,26 @@
+# Test run with N SWE-bench instances (default: 5)
+
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+BENCHMARKS_DIR="$(dirname "$(dirname "$SCRIPT_DIR")")"
+
+cd "$BENCHMARKS_DIR"
+
+# Load environment variables
+if [ -f .env ]; then
+    export $(cat .env | grep -v '#' | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)